keps/sig-network/2079-network-policy-port-range/README.md

```diff
@@ -280,13 +280,22 @@ _This section must be completed when targeting beta graduation to a release._
   an object specifying the range and validating if the traffic is allowed within 
   the specified range
 
+* **How can someone using this feature know that it is working for their instance?
+  - [x] Other
+   - Details:
+      The API Field must be present when a NetworkPolicy is created with that field.
+      The feature working correctly depends on the CNI implementation, so the operator can
+      look into CNI metrics to check if the rules are being applied correctly, like Calico 
+      that provides metrics like `felix_iptables_restore_errors` that can be used to 
+      verify if the amount of restoring errors raised after the feature being applied.
+ 
```

Comment:
Can you please also add that eventually we need to also add Status field to NetworkPolicy?